front/lib/api/auth0.ts

```diff
@@ -77,43 +124,46 @@ async function verifyAuth0Token(accessToken: string): Promise<jwt.JwtPayload> {
       },
       (err, decoded) => {
         if (err) {
-          reject(err);
-          return;
+          return resolve(new Err(err));
         }
         if (!decoded || typeof decoded !== "object") {
-          reject(new Error("No token payload"));
-          return;
+          return resolve(new Err(Error("No token payload")));
         }
-        resolve(decoded);
+
+        if (!isAuth0Payload(decoded)) {
+          logger.error("Invalid token payload.");
+          return resolve(new Err(Error("Invalid token payload.")));
+        }
+
+        const now = Math.floor(Date.now() / 1000);
```

Comment:
IIRC, jwt.verify takes care of this already.

Reply:
https://github.com/auth0/node-jsonwebtoken/blob/master/README.md#jwtverifytoken-secretorpublickey-options-callback